fee_allocator/accounting/fee_pipeline.py

```diff
@@ -45,6 +45,9 @@ def run_fees(
     core_pools["polygon"].pop(
         "0x65fe9314be50890fb01457be076fafd05ff32b9a000000000000000000000a96"
     )
+    core_pools["mainnet"].pop(
```

Comment:
Can we just remove this from the core pools json in main instead of building in an exception here, or if we are going to build an exception system make it based of lists visable in config/constants?

Reply:
This thing is happening almost every week and it's hard to follow. I will just leave it for now until we fix core pools list

Reply:
The core pools list should be the thing that chages a lot, not the code.  Did you create that PR at least?